IdentityCore/src/util/ios/MSIDAppExtensionUtil.h

```diff
@@ -33,14 +33,24 @@
 
 /// Determine whether or not the host app is an application extension based on the main bundle path
 + (BOOL)isExecutingInAppExtension;
+
+/// Application extension safe replacement for `openURL:`. The caller should make sure `isExecutingInAppExtension == NO` before calling this method.
++ (void)sharedApplicationOpenURL:(nonnull NSURL *)url;
```

Comment:
this header is in the iOS folder. it will need to move out to the top level folder now.

Reply:
Yes, Will changing the path cause issues? Both mac and iOS test apps seemed to build & run fine when I moved it.

Reply:
it won't work with CocoaPods which filters by path. Shouldn't cause any issues - you just might need to update broker and MSAL to use the new path.

Reply:
Moved header file to src/util. Pointed 'MSAL' pod in MSAL sample app projects (iOS & macOS) to my local MSAL project with these changes, it seemed to include MSIDAppExtensionUtil.h